docs/core/deploying/trimming-options.md

```diff
@@ -123,3 +123,37 @@ Symbols will normally be trimmed to match the trimmed assemblies. You can also r
     Remove symbols from the trimmed application, including embedded PDBs and separate PDB files. This applies to both the application code and any dependencies that come with symbols.
 
 The SDK also makes it possible to disable debugger support using the property `DebuggerSupport`. When debugger support is disabled, trimming will remove symbols automatically (`TrimmerRemoveSymbols` will default to true).
+
+## Trimming framework library features
+
+Several feature areas of the framework libraries come with linker directives that make it possible to remove the code for disabled features.
+
+- `<DebuggerSupport>false</DebuggerSupport>`
+
+    Remove dependencies that enable better debugging experiences. This will also [remove symbols](#Removing-symbols).
```

Comment:
Is it weird to say "dependencies" here? I guess I would have expected it to say

```suggestion
    Removes code that enables better debugging experiences. This will also [remove symbols](#Removing-symbols).
```